### PR TITLE
Create adapter with custom port

### LINKED ISSUE #131: Creating adapter with custom serialport
Creating adapter with custom serialport name, that not tests for manufacturer (SEGGER) and serialnumber.

### PR DESCRIPTION
**Pull request by pokornyib**
fix #131

### CODE REVIEW

File: api/adapterFactory.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ const Adapter = require('./adapter');
 const logLevel = require('./util/logLevel');
 const EventEmitter = require('events');
 
-const _bleDrivers = { v2: _bleDriverV2, v3: _bleDriverV3 };
+const _bleDrivers = {v2: _bleDriverV2, v3: _bleDriverV3};
```
Comment (bihanssen) [inline api/adapterFactory.js]: Keep the spaces, eslint rule.

File: api/adapterFactory.js [hunk 2]
```diff
@@ -291,6 +291,31 @@ class AdapterFactory extends EventEmitter {
             }
         });
     }
+
+    /**
+     * Create Adapter with on custom serialport
+     *
+     * @param sdVersion Softdevice version: 'v2' or 'v3'.
```
Comment (bihanssen) [inline api/adapterFactory.js]: The correct description would be "Softdevice API version", as the API version and the Softdevice version are not necessarily synchronized.

File: api/adapterFactory.js [hunk 3]
```diff
@@ -291,6 +291,31 @@ class AdapterFactory extends EventEmitter {
             }
         });
     }
+
+    /**
```
Comment (bihanssen) [inline api/adapterFactory.js]: For proper jsdoc syntax, add param types, e.g. @param sdVersion {string}

File: api/adapterFactory.js [hunk 4]
```diff
@@ -291,6 +291,31 @@ class AdapterFactory extends EventEmitter {
...
+     * Create Adapter with on custom serialport
```
Comment (bihanssen) [inline api/adapterFactory.js]: Typing error: "with on"

**Comment by pokornyib** [comment]
I have just done the requested changes, please review and merge it to master.

**Pull request merged**